docs/spelling_wordlist.txt

```diff
@@ -96,19 +97,19 @@ Rackspace
 Redis
 redis
 resize
-routable
 roadmap
+routable
 screenshot
 snapshotting
 stateful
 sysadmin
 syslog
 terabytes
-timestamp
 terraform
 tfvars
-unauthorised
+timestamp
 Ubuntu
+unauthorised
```

Comment:
I can't find any instance of authorised in the docs, so I've removed this. Was probably an old fix.